toml-config/src/config/parsers/action.rs

```diff
@@ -143,7 +144,11 @@ impl ActionParser<'_> {
 
     fn parse_show_workspace(&mut self, ext: &mut Extractor<'_>) -> ParseResult<Self> {
         let name = ext.extract(str("name"))?.value.to_string();
-        Ok(Action::ShowWorkspace { name })
+        let move_pointer = match ext.extract(opt(bol("move-pointer")))? {
```

Comment:
You should only call `extract` once. See other places in this file for how to extract multiple fields at once.